test/dlc_tests/ec2/mxnet/inference/test_mxnet_inference.py

```diff
@@ -2,14 +2,14 @@
 import pytest
 
 from test import test_utils
-from test.test_utils import CONTAINER_TESTS_PREFIX
+from test.test_utils import CONTAINER_TESTS_PREFIX, get_framework_and_version_from_tag
 from test.test_utils.ec2 import get_ec2_instance_type, execute_ec2_inference_test, get_ec2_accelerator_type
 from test.dlc_tests.conftest import LOGGER
 
 
 SQUEEZENET_MODEL = "squeezenet"
 BERT_MODEL = "bert_sst"
-RESNET_EIA_MODEL = "resnet-152-eia"
+DEFAULT_RESNET_EIA_MODEL = "resnet-152-eia"
```

Comment:
The model name is also used in [L68](https://github.com/tejaschumbalkar/deep-learning-containers/blob/master/test/dlc_tests/ec2/mxnet/inference/test_mxnet_inference.py#L68) for `get_inference_run_command` function. 

Can we resolve the model name in the pytest function itself and pass it to the `run_ec2_mxnet_inference` function or some other way?

Reply:
sounds like a plan. Will resolve the model name inside the pytest functions